src/main/java/uk/gov/digital/ho/hocs/audit/export/ExportService.java

```diff
@@ -539,6 +541,38 @@ public void staticUserExport(OutputStream output, boolean convertHeader) throws
         }
     }
 
+    void userWithTeamsExport(OutputStream output) throws IOException{
+        log.info("Exporting USER_TEAMS_DATA to CSV", value(EVENT, CSV_EXPORT_START));
+        OutputStream buffer = new BufferedOutputStream(output);
+        OutputStreamWriter outputWriter = new OutputStreamWriter(buffer, "UTF-8");
+        List<String> headers = Stream.of("User UUID", "Teams UUID").collect(Collectors.toList());
+
+        Set<UserWithTeamsDto> users = infoClient.getUsersWithTeams();
+
+        try (CSVPrinter printer = new CSVPrinter(outputWriter, CSVFormat.DEFAULT.withHeader(headers.toArray(new String[0])))) {
+            users.forEach((user) -> {
+                try {
+                    List<UUID> userTeams = user.getTeamUUIDs();
+                    StringBuilder teamUUIDs = new StringBuilder();
```

Comment:
We can get rid of this through the following: 

`userTeams.stream().map(UUID::toString).collect(Collectors.joining(", "));`